test/AdaptivityTests/mytests.jl

```diff
@@ -0,0 +1,89 @@
+module MyTests
```

Comment:
Should this file be here?

Reply:
It should not, sorry. Those where my exploratory tests... I removed them now.